easydita-knowledge-portal/functions.php

```diff
@@ -148,9 +148,9 @@ function easydita_knowledge_portal_get_hierarchy() {
 		$hierarchy = array();
 		if($post->ID) {
 			$hierarchy[] = $post->ID;
-			
```

Comment:
Discovered what the issue was: my text editor removes trailing whitespace by default.